README.md

```diff
@@ -131,10 +131,12 @@ Since Lob charges per request, you can set `delaySearch` to true which delays an
 | Introduction of Autocomplete Forms |
 | Removed UI restrictions blocking flow of autocomplete results | 
 ### 1.0.7
+| 1.0.7 |
```

Comment:
Our package.json updates to 1.0.10, did we goof up somewhere and forget to document release logs for 1.0.8 and 1.0.9?

Reply:
yeah I'm partially to blame for that one lol, ran npm publish a couple times extra